### PR TITLE
Change Google Translate info

### PR DESCRIPTION
**Pull request by dveeden**
### What is changed, added or deleted? (Required)

- Add German and Spanish
- Translate the website instead of the docs markdown

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,9 +13,11 @@ Currently, the official documentation supports two languages:
 
 You can use Google Translate to view the documentation in different languages. For example:
 
-- `fr`: [documentation in French](https://translate.google.com/translate?hl=en&sl=en&tl=fr&u=https%3A%2F%2Fgithub.com%2Fpingcap%2Fdocs%2Fblob%2Fmaster%2FTOC.md)
-- `ja`: [documentation in Japanese](https://translate.google.com/translate?hl=en&sl=en&tl=ja&u=https%3A%2F%2Fgithub.com%2Fpingcap%2Fdocs%2Fblob%2Fmaster%2FTOC.md)
-- `ko`: [documentation in Korean](https://translate.google.com/translate?hl=en&sl=en&tl=ko&u=https%3A%2F%2Fgithub.com%2Fpingcap%2Fdocs%2Fblob%2Fmaster%2FTOC.md)
+- `fr`: [documentation in French](https://docs-pingcap-com.translate.goog/tidb/stable?_x_tr_sl=auto&_x_tr_tl=fr&_x_tr_hl=en&_x_tr_pto=wapp&_x_tr_hist=true)
+- `ja`: [documentation in Japanese](https://docs-pingcap-com.translate.goog/tidb/stable?_x_tr_sl=auto&_x_tr_tl=ja&_x_tr_hl=en&_x_tr_pto=wapp&_x_tr_hist=true)
```
Comment (ran-huang) [inline README.md:17]: None of these links seems to work, and contents remain untranslated. I'm not sure if it's just my network issue. 

image
image

Comment (dveeden) [reply README.md:17]: What browser are you using? This works for me on Firefox 121.0.1 and Chrome 120.0.6099.224

Comment (dveeden) [reply README.md:17]: Maybe you can try going to https://translate.google.com and then enter `https://docs.pingcap.com/tidb/stable/overview` in the left box and select something like "Spanish" on the right and then click the URL in the right box?

Comment (dveeden) [reply README.md:17]: Another option is https://addons.mozilla.org/en-US/firefox/addon/firefox-translations/ but that's browser specific and can't be integrated into the readme as easily.

Comment (dveeden) [reply README.md:17]: image

If you open the network tab of the developer console and filter on `translate.googleapis.com/translate` do you see something like this?

Comment (ran-huang) [reply README.md:17]: Yes. I can see the request fail. It happens when I don't use a global VPN to bypass the GFW. 
image

With a VPN it's fine:

image

**Pull request merged**